jupyterlab/commands.py

```diff
@@ -476,12 +477,13 @@ def read_package(target):
 
 class _AppHandler(object):
 
-    def __init__(self, app_dir, logger=None, kill_event=None):
+    def __init__(self, app_dir, logger=None, kill_event=None, core_config=None):
         """Create a new _AppHandler object
         """
         self.app_dir = app_dir or get_app_dir()
         self.sys_dir = get_app_dir()
         self.logger = _ensure_logger(logger)
+        self.core_data = (core_config or CoreConfig()).data
```

Comment:
@vidartf I mean that we are using the data here in the app handler.

Reply:
Doesn't the docstring of `data` cover this use?